### PR TITLE
Watch bookmarks to beta 

### PR DESCRIPTION
**Pull request by wojtek-t**
Ref: https://github.com/kubernetes/website/pull/15530#issue-301067619

/assign @lavalamp @deads2k

### CODE REVIEW

File: content/en/docs/reference/using-api/api-concepts.md [hunk 1]
```diff
@@ -104,7 +104,7 @@ To mitigate the impact of short history window, we introduced a concept of `book
           "object": {"kind": "Pod", "apiVersion": "v1", "metadata": {"resourceVersion": "12746"} }
         }
 
-`Bookmark` events can be requested by `allowWatchBookmarks=true` option in watch requests, but clients shouldn't assume bookmarks are returned at any specific interval, nor may they assume the server will send any `bookmark` event. As of 1.15 release, it is an Alpha feature.
+`Bookmark` events can be requested by `allowWatchBookmarks=true` option in watch requests, but clients shouldn't assume bookmarks are returned at any specific interval, nor may they assume the server will send any `bookmark` event. As of 1.16 release, it is a Beta feature.
```
Comment (kbhawkey) [inline content/en/docs/reference/using-api/api-concepts.md]: hello @wojtek-t. The updates seem fine. What do you think about using the feature-state shortcode, such as
{{< feature-state for_k8s_version="v1.16" state="beta" >}}, for the watch/bookmark feature? As it stands, the feature state and version are hard to locate in the paragraph.
Also, should this feature/state be listed in the feature gates table,
https://5d397c09ad55c5000cd80afd--kubernetes-io-vnext-staging.netlify.com/docs/reference/command-line-tools-reference/feature-gates/

Comment (wojtek-t) [reply content/en/docs/reference/using-api/api-concepts.md]: @kbhawkey - thanks for suggestion. I applied the comment.

PTAL if the current state is what you expected.

Comment (kbhawkey) [reply content/en/docs/reference/using-api/api-concepts.md]: hello, @wojtek-t  
one nit: use back ticks around boolean flag, Default column, in feature gates table, for example:
`true`
https://5d3b57ae198fa600097ff5b0--kubernetes-io-vnext-staging.netlify.com/docs/reference/command-line-tools-reference/feature-gates/

File: content/en/docs/reference/using-api/api-concepts.md [hunk 2]
```diff
@@ -87,6 +87,8 @@ For example:
         ...
 
 A given Kubernetes server will only preserve a historical list of changes for a limited time. Clusters using etcd3 preserve changes in the last 5 minutes by default.  When the requested watch operations fail because the historical version of that resource is not available, clients must handle the case by recognizing the status code `410 Gone`, clearing their local cache, performing a list operation, and starting the watch from the `resourceVersion` returned by that new list operation. Most client libraries offer some form of standard tool for this logic. (In Go this is called a `Reflector` and is located in the `k8s.io/client-go/cache` package.)
+
+{{< feature-state for_k8s_version="v1.16" state="beta" >}}
```
Comment (kbhawkey) [inline content/en/docs/reference/using-api/api-concepts.md]: another nit, totally optional:
You could add a sub-header, such as:
### Watch bookmarks

And then below the sub-header add the feature-state shortcode, followed by a new line and the rest of the content (see if this looks ok).

Comment (wojtek-t) [reply content/en/docs/reference/using-api/api-concepts.md]: done

**Comment by wojtek-t** [comment]
@kbhawkey @liggitt - PTAL

**Comment by zacharysarah** [comment]
@wojtek-t 👋 Please rebase this PR in order to continue. Looks like it's headed in the right direction, so anyone can `/lgtm` after rebase.

/approve

**Comment by wojtek-t** [comment]
Rebased.

/hold cancel

PTAL

**Comment by zacharysarah** [comment]
@wojtek-t Thanks for rebasing. Sorry, I missed that this is for 1.16; I'll leave approval for @simplytunde. 

/approve cancel
/lgtm

**Comment by zacharysarah** [comment]
Screen Shot 2019-08-07 at 12 46 10 PM

> /approve cancel
> 
> [APPROVALNOTIFIER] This PR is APPROVED

/shrug

**Comment by zacharysarah** [comment]
Screen Shot 2019-08-07 at 12 46 10 PM

> /approve cancel
> 
> [APPROVALNOTIFIER] This PR is APPROVED

/shrug

@kubernetes/sig-testing-pr-reviews 👋 Did I miss something with removing `/approve`?

**Comment by zacharysarah** [comment]
It looks like a force push caught a bunch of commits that triggered non-applicable labels. I'm removing them with magic admin powers.

**Comment by wojtek-t** [comment]
@zacharysarah - can we hold cancel now?

**Comment by zacharysarah** [comment]
@wojtek-t Because this PR is part of the 1.16 release cycle, it's up to the docs release lead (@simplytunde) as to when it merges.

That said, it LGTM.

**Pull request merged**